app/models/payment.rb

```diff
@@ -40,6 +40,12 @@ def payment_date
     fee_pay_info && Time.strptime(fee_pay_info['paymentDate'], '%Y-%m-%d')
   end
 
+  def sort_key
+    return Time.zone.parse(fee_pay_info['paymentDate']) if fee_pay_info && fee_pay_info['paymentDate']
+
+    Request::END_OF_DAYS
```

Comment:
Should the payments in history w/o a payment date sort at the top or the bottom?

Reply:
We've sorted missing data to the top everywhere else.

Reply:
@jkeck can you clarify and provide an example user?

Reply:
I don't have an example handy, but my question/concern comes from how some of the accessors in `Payment` are written.

https://github.com/sul-dlss/mylibrary/blob/ddd8205559a6c61733c973543ba318971d32378b/app/models/payment.rb#L39-L41

Since this method could return `nil` and sorting will compare a `Time` vs. `nil` (which would throw an error) so I'm guarding against that.